Student/StudentUITests/Login/LoginStartTests.swift

```diff
@@ -41,4 +41,9 @@ class LoginStartTests: StudentUITestCase {
 
         LoginWeb.webView.waitToExist()
     }
+
+    // HACK: relaunch to avoid a double-view bug
+    func testZzz() {
+        launch()
+    }
```

Comment:
sad that we need this, but better that the culprit here have this hack, than whatever comes next get screwed.